Framework/API/inc/MantidAPI/MatrixWorkspace.h

```diff
@@ -15,6 +15,7 @@
 #include "MantidKernel/EmptyValues.h"
 
 #include <atomic>
+#include <boost/optional.hpp>
```

Comment:
Tiny thing but I don't think we need this header anymore as we are using `atomic<bool>`?